src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -87,6 +87,9 @@
     /** Map of third party Checkstyle module names to the set of their fully qualified names. */
     private Map<String, Set<String>> thirdPartyNameToFullModuleNames;
 
+    /** If true old style force loading will be used. */
+    private boolean forceLoad;
```

Comment:
please rename it to "useBruteForceModuleLoading".

Put in javadoc reference that "it is required for eclipse-cs plugin"